Tools/CISupport/ews-build/steps.py

```diff
@@ -3372,10 +3376,10 @@ class TransferToS3(master.MasterShellCommand):
     name = 'transfer-to-s3'
     description = ['transferring to s3']
     descriptionDone = ['Transferred archive to S3']
-    archive = WithProperties('public_html/archives/%(fullPlatform)s-%(architecture)s-%(configuration)s/%(patch_id)s.zip')
+    archive = WithProperties('public_html/archives/%(fullPlatform)s-%(architecture)s-%(configuration)s/%(change_id)s.zip')
```

Comment:
Note that we would need to check if our S3 configuration need any update for this or not (for PR workflow).

Reply:
What is the right way to verify our S3 configuration? The UAT instance doesn't send builds to S3, right?

Reply:
From amazon console, I will check that.
Yes, uat instance (or any testing instance) do not send data to S3.
We can land this change though, but need to verify that before we start builder builds for PR workflow on production EWS instance.